gsa/public/locales/gsa-de.json

```diff
@@ -1,4 +1,6 @@
 {
+  "\nThis could have the following reasons:\n1. You might have followed an incorrect link and the CVE does not exist\n2. The CVE might not be included in the SCAP database yet. For new CVEs it can take a month or more until they become available.": "\nDies kann folgende Ursachen haben:\n1. Möglicherweise war der Link für diese CVE inkorrekt und sie existiert nicht.\n2. Die CVE ist vielleicht noch nicht in der SCAP-Datenbank enthalten. Für neue CVEs kann es einen Monat oder länger dauern bis sie verfügbar sind.",
+  "\nYou might have followed an incorrect link and the {{entity}} does not exist.": "Möglicherweise war der Link für diese Ressource inkorrekt und sie existiert nicht.",
```

Comment:
This translation doesn't really fit not does it contain the {{entity}}

Reply:
It doesn't contain the entity to prevent using "der/die/das" in the German translation, as we have no idea what kind of entity it will be.
This translation was already in GSA before it was lost with the merge that Sarah mentioned.

Reply:
Happy for any better translation :)

Reply:
Should we ask @k-schlosser?

Reply:
Done. We agreed on leaving the entity out because of the article issue and k-schlosser approximated the first part better to the English original.